index.bs

```diff
@@ -3435,6 +3441,150 @@ The [=remote end steps=] with <var ignore>session</var> and |command parameters|
 
 </div>
 
+#### The browsingContext.print Command ####  {#command-browsingContext-print}
+
+The <dfn export for=commands>browsingContext.print</dfn> command
+creates a paginated representation of a document, and returns it as a
```

Comment:
Shall we mention that it actually `renders` the content into a paginated format similar to what we have in classic? Especially including that it is in PDF format would be helpful.

Reply:
I was considering that PDF is an implementation detail, and that we could support other formats in the future. But I guess if that happens we can update the text.